test/test-applications/azure-functions/Samples.AzureFunctions.AllTriggers/local.settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "DefaultEndpointsProtocol=http;AccountName=devstoreaccount1;AccountKey=Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw==;BlobEndpoint=http://127.0.0.1:10000/devstoreaccount1;",
```

Comment:
Is this the built-in account for the local blob storage? Just double-checking credientials that are being checked in

Reply:
Yes this is built in, no secrets here